test/phoenix_live_view/integrations/event_test.exs

```diff
@@ -0,0 +1,70 @@
+defmodule Phoenix.LiveView.EventTest do
+  use ExUnit.Case, async: false
```

Comment:
`async: true` should be fine? If it cannot be async: true, please drop a comment at the top detailing why. :)